app/views/home/businesses.html.erb

```diff
@@ -35,14 +35,14 @@
 
   <div class="col-sm-6 col-md-12 float-left float-md-none pl-sm-4 pl-md-0 clearfix">
     <div class="col-12 col-md-6 float-md-right pl-0 pl-md-3 mb-3 mb-sm-0">
-      <h3 class="alt-h3 mb-1">Improve your technology</h3>
+      <h3 class="alt-h3 mb-1">Reduce overhead</h3>
```

Comment:
Yes, and you convinced me. :smile: I think to use Erin's phrase, it's a proactive investment: if you submit changes upstream, you're still doing the same amount of work right now, but less work in the future.